### PR TITLE
Update preview.md

### PR DESCRIPTION
**Pull request by msbemba**
Added required permissions

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4005

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/security/mtp/preview.md [hunk 1]
```diff
@@ -38,6 +38,15 @@ Learn about new features in the Microsoft 365 Defender preview release and be am
 
 For more information on new capabilities that are generally available, see [What's new in Microsoft 365 Defender](whats-new.md).
 
+## Required permissions
+
+Accounts assigned the following Azure Active Directory (AD) roles can Turn On  Microsoft 365 Defender Preview features
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/preview.md]:
```suggestion
Accounts assigned the following Azure Active Directory (Azure AD) roles can turn on  Microsoft 365 Defender Preview features:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks.

**Pull request merged**